README.md

```diff
@@ -1,2 +1,149 @@
+
+Be a Monica, not a Rachel.
+
+<iframe src="https://giphy.com/embed/26FPxK7LVhEHammiY" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/friends-nick-at-nite-26FPxK7LVhEHammiY"></a></p>
+
+<!-- ## Decisions, Decisions
+
+There are some major decisions that you must make when you are planning a project. Making these decisions is a key part of iteratively developing your architecture design.
+
+You need to:
+
+- Determine the Application Type
+- Determine the Deployment Strategy
+  _"Your application may be deployed in a variety of environments, each with its own specific set of constraints such as physical separation of components across different servers, a limitation on networking protocols, firewall and router configurations, and more."_
+- Determine the Appropriate Technologies
+- Determine the Quality Attributes
+  _e.g. run-time qualities, security, performance, usability, scalability_
+- Determine the Crosscutting Concerns
+  _Where can you use centralised solutions to problems that are specific to more than one layer of your application?_
+
+-- "Key Design Considerations", from the [Microsoft Application Architecture Guide](https://msdn.microsoft.com/en-us/library/ee658124.aspx#DesignConsiderations) -->
+
+## Key Principles To Keep in Mind
+
+We're going to start designing the architecture for our own app in a bit.
+
+When you are architecting an app, try to keep these key principles in mind:
+
+- How can you separate your concerns? i.e. keep unrelated things in separate sections
+- Which aspects of your app are generic, and which are specific to your requirements?
+- How can you make it easy to test?
+- How can you make it easy to change?
+- Can you anticipate changes that are likely to occur?
+
+Take a few minutes to look at your Week 2 project and come up with answers to these questions:
+
+- Would you change the architecture of your app based on your answers to these questions?
+- What would your new file structure look like?
+
+If you finish early, [here](https://msdn.microsoft.com/en-us/library/ee658124.aspx#KeyDesignPrinciples) are some more detailed Key Principles.
+
+## Group Exercise: Whiteboard Your Architecture
+
+<iframe src="https://giphy.com/embed/l2SpKy09apbIwmXVS" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/southparkgifs-l2SpKy09apbIwmXVS"></a></p>
+
+There are 2 main reasons why you should whiteboard your architecture:
+
+- If you cannot whiteboard the architecture then it suggests that it is not well understood.
+- If you can provide a clear and concise whiteboard diagram, others will understand it and you can communicate details to them more easily.
+
+We have already drawn some architectural diagrams together.
+
+Now it's time to get into  your project groups Again. We're going to architect a hypothetical app together.
```

Comment:
Think the instructions should be a little ore direct here. (ie 'draw out a flow diagram that represents a suitable app architecture and the flow or communication as described in the weather app below)

ALSO - I think an important follow-on exercise as soon as they've done that would be to write up a proposed file structure plan that would suit the architecture that they've drawn out.
ie:
``` md

root/
  index.html
  assets/
    css/
      index.css
    scripts/
      index.js
      location_request.js
      forecast_request.js
```

IMO the diagram greatly serves the design of the file structure, and the two plans are best done together.